.github/workflows/documentation-preview.yml

```diff
@@ -0,0 +1,49 @@
+name: Documentation Preview Generator
+
+on:
+  workflow_run:
+    workflows: [Documentation Preview Request]
+    types:
+      - completed
+
+env:
+  PREVIEW_PUBLISH_USERNAME: ${{ secrets.PREVIEW_PUBLISH_USERNAME }}
+  PREVIEW_PUBLISH_REPOSITORY: ${{ secrets.PREVIEW_PUBLISH_REPOSITORY }}
+  PREVIEW_PUBLISH_BRANCH: ${{ secrets.PREVIEW_PUBLISH_BRANCH }}
+  PREVIEW_PUBLISH_TOKEN: ${{ secrets.PREVIEW_PUBLISH_TOKEN }}
```

Comment:
except token do we really need those as part of secret, might be just env could work?

Reply:
You're right. Apart from secret other options would work just fine as env. Shall I update it ?

Reply:
please update. can even be defined in the top of the workflow

Reply:
I've added these variables as clear text:
- Preview Publish GitHub Org Name : `crc-org`
- Preview Publish GitHub repo name: `crc-docs-gh-action-generated-preview`
- Preview Publish GitHub branch: `gh-pages`